src/elements/hx-accordion-panel/index.spec.js

```diff
@@ -0,0 +1,148 @@
+                const mode = component.shadowRoot.mode;
+
+                expect(mode).to.equal('open');
+            });
+
+            it('should have a single <slot>', async () => {
+                const component = /** @type { HXAccordionPanelElement } */ await fixture(template);
+                const shadow = component.shadowRoot;
+                const query = shadow.querySelectorAll('slot');
+                const len = query.length;
+
+                expect(len).to.be.equal(2);
+            });
+
+            it('should have an unnamed <slot>', async () => {
+                const component = /** @type { HXAccordionPanelElement } */ await fixture(template);
+                const name = component.slot;
+
+                if ( name !== null ) {
+                    expect(name).to.be.equal('');
+                } else {
+                    expect(name).to.be.null;  // IE11, Legacy Edge, and older browsers
+                }
+            });
+        });
+
+        describe('verify Shadow DOM markup', () => {
+            it('markup should contain a #hxToggle button', async () => {
+                const elSelector = 'button#hxToggle';
+                const id = 'hxToggle';
+                const component = /** @type { HXAccordionPanelElement } */ await fixture(template);
+                const shadow = component.shadowRoot;
+                const query = shadow.querySelector(elSelector);
+                const queryId = query.id;
+
+                expect(queryId).to.equal(id);
+            });
+
+            it('markup should contain a <hx-icon> with an angle-down type <hx-icon>', async () => {
+                const elSelector = 'button#hxToggle > div > hx-icon';
+                const iconType = 'angle-down';
+                const component = /** @type { HXAccordionPanelElement } */ await fixture(template);
+                const shadow = component.shadowRoot;
+                const query = shadow.querySelector(elSelector);
+                const queryId = query.type;
+
+                expect(queryId).to.equal(iconType);
+            });
+
+            it('markup should contain a #hxBody <div>', async () => {
+                const elSelector = 'div#hxBody';
+                const id = 'hxBody';
+                const component = /** @type { HXAccordionPanelElement } */ await fixture(template);
+                const shadow = component.shadowRoot;
+                const query = shadow.querySelector(elSelector);
+                const queryId = query.id;
+
+                expect(queryId).to.equal(id);
+            });
+        });
+    });
+
+    describe('test $onConnect method', () => {
+        it('should be able to open', async () => {
+            const component = /** @type {HXAccordionPanelElement} */ await fixture(template);
+            component.open = true;
+            const attr = component.hasAttribute('open');
+
+            expect(attr).to.be.true;
+        });
```

Comment:
You can remove this test.  It's a duplicate.

Reply:
Changes have been done. Please review